examples/README.md

```diff
@@ -182,6 +182,35 @@ example.
 
    </details>
 
+### Basic Usage
```

Comment:
Maybe we should merge the `exported-function` item with this tables and memories items, into a single section named Externals or Externs, I don't know, something like that. Also don't forget to add links to the example files :-).